dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -288,7 +288,7 @@ class updatedns
         if ($this->_dnsVerboseLog) {
             log_error("Dynamic DNS ({$this->_dnsHost}): running dyndns_failover_interface for {$dnsRequestIf}. found {$this->_dnsRequestIf}");
         }
-        $this->_dnsRequestIfIP = $this->_useIPv6 ? get_interface_ipv6($this->_dnsRequestIf) : get_interface_ip($this->_dnsRequestIf);
+        $this->_dnsRequestIfIP = $this->_dnsRequestIf;
```

Comment:
Hmm, this makes _dnsRequestIfIP totally obsolete and affects ALL services rather than just GoDaddy. Can you try c8a115291df30e46ac0b instead?

Reply:
@fichtner - That appears to work fine.